controllers/provider-openstack/charts/images.yaml

```diff
@@ -13,4 +13,8 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "0.6.4"
\ No newline at end of file
+  tag: "0.6.4"
+- name: openstack-cloud-controller-manager
```

Comment:
Can you replace `hyperkube` with this? I guess `hyperkube` is no longer needed now? If not, please remove all occurrences of `hyperkube` also in the code of the openstack controller.